controller/src/main/java/io/pravega/controller/server/rest/generated/api/ApiException.java

```diff
@@ -1,6 +1,6 @@
 package io.pravega.controller.server.rest.generated.api;
 
-@SuppressWarnings("all")
```

Comment:
Change from automatic codegen (unsure why this line was removed).

Reply:
> So why not add it back? It's not related, so revert the file.

The reason why I did not revert the change right away was to signal that these changes occur every time the swagger code gen is executed. They change so little I guess it doesn't matter but I was wondering if this annotation is now unnecessary or there is a 'fix' that includes it in the generation.